pkg/kubernetes/deployment/appsv1/v1alpha1/kubernetes.go

```diff
@@ -394,6 +418,30 @@ func (k *Kubeclient) Create(deployment *appsv1.Deployment) (*appsv1.Deployment,
 	return k.create(cli, k.namespace, deployment)
 }
 
+// Update updates a deployment in specified namespace in kubernetes cluster
+func (k *Kubeclient) Update(deployment *appsv1.Deployment) (*appsv1.Deployment, error) {
+
+	if debug.EI.IsDeploymentCreateErrorInjected() {
```

Comment:
There is a `IsDeploymentUpdateErrorInjected`, in the debug package. I think that should be used.

Reply:
Missed it. Thanks :+1: